Revision

HunterX-7 opened the pull request:
In this Pull Request, I:
> Created the revision branch.

> Followed GitHub flow.

> Added the nav section to create the toolbar (header)

-  Added the h3 for the Welcome-icon

-  Added the img for the hamburger-icon

> Added the div section to create the headline section (right after the header)

-  Added the  h2 for the title

-  Added the p for the paragraph

-  Added another nav section for the socials 

-  Added the nav section for the scroll-down button 

- Added flexbox in all sections

- Added the font, colors, images, text, and margins to achieve a similar look to template #3


![Toolbar and headline-results](https://user-images.githubusercontent.com/90111138/198179881-579a863f-dd0d-4980-826a-6dcf9cbecbb6.png)

Santiago220991 commented on index.html:
```diff
@@ -0,0 +1,32 @@
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav class="header">
+        <h3>Welcome</h3>
+        <a href=""><img src="images/PNG/Enabled.png" alt="Hamburger-icon"></a>
+    </nav>
```
- Kindly replace the `nav` tag with a `header` tag. Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. :nerd_face: :nerd_face: 
- Please add an `a` tag to your logo. This is for three reasons: convention, semantics, and user experience(UX).  :pray: :pray: 
- Please add a `tag` to your hamburger button. This also is for semantics reasons. :+1: :smiley: :smiley:

Santiago220991 commented on index.html:
```diff
@@ -0,0 +1,32 @@
+        <h3>Welcome</h3>
+        <a href=""><img src="images/PNG/Enabled.png" alt="Hamburger-icon"></a>
+    </nav>
+    <div class="title">
+        <h2 class="title1">Hey there. I'm Francisco<br><span class="title2">I'm a software developer</span></br></h2>
+        <p>I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don't hesitate to contact me.</p>
+        <nav class="socials">
+            <a href="#"><img src="images/PNG/twitter.png" alt="twitter-icon"></a>
+            <a href="#"><img src="images/PNG/Linkedin icon.png" alt="linkedin-icon"></a>
+            <a href="#"><img src="images/PNG/medium.png" alt="medium-icon"></a>
+            <a href="#"><img src="images/PNG/github.png" alt="github-icon"></a>
+            <a href="#"><img src="images/PNG/angellist.png" alt="angellist-icon"></a>
+        </nav>
+        <nav>
+            <a class="action" href="#"><img src="images/PNG/Action.png" alt="down-arrow"></a>
+        </nav>
```
- Kindly replace the `div` tag for a `section` tag. It represents a generic standalone section of the document and should be wrapped inside a `section` tag. :nerd_face: :nerd_face:  
- Kindly replace the `h2` tag with `h1` tag. If you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag, please. :+1: :smiley: :smiley: 
- The use of `a` tags for the icons is correct however please replace the `nav` tag with a `ul` tag to group the icons, and use `li` tags per icon. This way will be semantically correct. :eyes: :eyes: 
- Kindly remove the `nav` tag on line `27` it is not necessary to wrap this element inside a `nav` tag.

Santiago220991 reviewed (changes requested):
Hi @HunterX-7,
I'm Santiago your reviewer for this PR. :wave: :wave:  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<img src="https://media3.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>

## Highlights
:heavy_check_mark: Followed GitHub flow.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Good README file.
:heavy_check_mark: Good PR summary.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
[OPTIONAL] Kindly make sure that your project is responsive until the desktop screen size. :eyes: :eyes. To do this you should not use fixed `with` values in your containers. 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HunterX-7 commented:
In this Pull Request, I made the following requested changes:

- Added the header tag containing the welcome logo and the hamburger button.
- Added an a tag to my logo and another to my hamburger button.
- Added the section tag containing a div tag, which contains my title, a p tag, an ul tag, and a button.
- Replaced the h2 tag with a h1 tag
- Added an ul tag to my social icons with their respective li tag for each of them.
- Removed the nav tag on line 27.

Optional suggestions:

- Adjusted the flexbox of my container to make them more responsive.
![Toolbar and headline-results](https://user-images.githubusercontent.com/90111138/198369844-cac1a9b0-d296-4637-b8e8-dd9adac0675b.png)

PinkMoon25 reviewed (approved):
Hi @HunterX-7 , 

Good work implementing the changes :clap: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<img src="https://media0.giphy.com/media/cOvgh3VjLmeg8LLBtk/giphy-downsized-medium.gif"/>

## Highlights
linters checks passing :heavy_check_mark: 
good commits :heavy_check_mark: 
Professional readme :heavy_check_mark: 
design follows figma template :heavy_check_mark: 
Used flexbox for layout :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
